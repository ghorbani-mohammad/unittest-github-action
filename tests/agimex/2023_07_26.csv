

"SCPAR TGSFS - GRAIN PIER # 101 - SPEC AND RESTRICTIONS","AGIMEX"


"Max Air Draft:","16.5 mts"
"Ship Loaders:","02"
"Loading rate:","1,500mt/h"


"Length (M) 240 mts","W. Density: 1018 to 1022 BW Local time: (UTC -3"


"Max Beam 40 mts","Max. sailing draft: 12.80 m. In/out night navigation: max. draft 11 m"


"Ship Loader Outreach: 35 mts (max)","","","","","","","1,200mt/hr (with 15° inclination angle)","","","","","","","","MARÍTIMA LIDA"


"Vessels (*)","ETA/ATA","ETA/ATB","ETS/ATS","Aw time (days)","Status","Charterer","Agency","Port Operator","Terminal","Cargo","POD/POL","Op Type","Pre-Plan (MT)","Ttl loaded","Cgo Balance"
"TORM BIRGITTE","21-07-23 19:24lt","26-07-23 09:45lt","28-07-23 09:00lt","4.60","Berthed","Bunge","Litoral","Litoral","BA","SBOIL","China/Malaysia","Load","26,300.000","","26,300.000"
"BULK GREECE","13-07-23 14:54lt","28-07-23","30-07-23 09:00lt","14.75","Waiting","C&D Xiamen","Alphamar","Zport","MG","SBS","China","Load","67,200.000","225m LOA",""
"BBG LIUZHOU","16-07-23 14:22lt","30-07-23 09:00If","01-08-23 09:00lt","13.78","Waiting","Cofco","Cargonave","Zport","MG","CORN","China","Load","67,600.000","225m LOA",""
"PANAGIA FORCE","16-07-23 21:00lt","01-08-23 09:00lt","03-08-23 09:00lt","15.50","Waiting","C.A.O.","Cargonave","Zport","MG","SBS","China","Load","66,000.000","225m LOA",""
"SOYA TIANJIN","21-07-23 10:48lt","03-08-23 09:00lt","05-08-23 09:00lt","12.93","Waiting","Bunge","Litoral","Litoral","BA+TG","SBS","Vietnam","Load","66,380.000","229m LOA",""
"KIM OLDENDORFF","11-07-23 04:00lt","05-08-23 09:00lt","07-08-23 09:00lt","25.21","Waiting","Cofco","Cargonave","Litoral","BA+TG","SBS","China","Load","65,000.000","229m LOA",""
"CALYPSO V","22-07-23 14:00lt","07-08-23 09:00lt","09-08-23 09:00lt","15.79","Waiting","BTG Pactual","Alphamar","Soin","TG","SBS","China","Load","55,000.000","225m LOA",""
"ANGLO BARINTHUS","27-07-23 16:00lt","09-08-23 09:00If","11-08-23 09:00lt","12.71","Expected","Olam","Cargonave","Zport","MG","SBS","China","Load","68,250.000","229m LOA",""
"GREEN SAPHIRE","28-07-23 02:00lt","11-08-23 09:00If","12-08-23 09:00lt","14.29","Expected","Enerfo","Alphamar","Zport","MG","CORN","Malaysia","Load","23,000.000","210m LOA","Top-off"
"KAPTA DIMITRIS A","31-07-23 00:00lt","12-08-23 09:00lt","14-08-23 09:00lt","12.38","Expected","Bunge","Litoral","Litoral","BA+TG","SBS","China","Load","66,000.000","229m LOA",""
"GENEVA STAR","01-08-23 01:00lt","14-08-23 09:00lt","16-08-23 09:00lt","13.33","Expected","Bunge","Litoral","Litoral","BA+TG","CORN","China","Load","65,000.000","229m LOA",""
">> SHED 101 INTERDICTION","","16-08-23 09:00lt","19-08-23 09:00lt","","Exchange of","ship's fenders","ordered by","Port Authority","No vessels","scheduled","during this period","","","",""
"BBG BRIGHT","03-08-23 00:00lt","19-08-23 09:00lt","21-08-23 09:00lt","16.38","Expected","Cofco","Cargonave","Zport","MG","CORN","China","Load","67,600.000","229m LOA",""
"BLUE AKIHABARA","05-08-23 00:00lt","21-08-23 09:00lt","23-08-23 09:00lt","16.38","Expected","Bunge","Litoral","Litoral","BA+TG","CORN","China","Load","62,000.000","200m LOA",""
"KEY JOURNEY","07-08-23 00:00lt","23-08-23 09:00lt","25-08-23 09:00lt","16.38","Expected","Cofco","Cargonave","Zport","MG","CORN","China","Load","66,000.000","225m LOA",""
"WISDOM DIVA","08-08-23 00:00lt","25-08-23 09:00lt","27-08-23 09:00lt","17.38","Expected","C.A.O.","Alphamar","Zport","MG","SBS","China","Load","66,000.000","229m LOA",""
"BRAVEHEART I","10-08-23 00:00lt","27-08-23 09:00lt","29-08-23 09:00lt","17.38","Expected","Bunge","Litoral","Litoral","BA+TG","CORN","China","Load","65,000.000","229m LOA",""
"SAKIZAYA BRAVE","10-08-23 00:00lt","29-08-23 09:00lt","31-08-23 09:00lt","19.38","Expected","Cofco","Cargonave","Zport","MG","SBS","China","Load","66,000.000","225m LOA",""
"","","Average","waiting time (in days)","14","","","","","Ttl cargo","estimated:","1,028,330.000","MT","","Cargo","Terminals:"


"BUNGE S/A","*BA*"
"SCPAR/TGSFS ==>","*TG*"
"MGB/TERLOGS","*MG*"


Remarks:,,

The general criterion for the definition of berthings will be the chronological order of ship's arrival in outer anchorage area.,,

Ship's ATA will be informed by official document sent by Ship's Captain to his local Agent, and in case of doubt, Port Auhority may request a copy of the Notice of Readiness (NOR) or the Automatic Information System (AIS) record,,

of the Pilot Station.,,

(*) Please stay aware that port line-up is subject to sudden, unexpected/unforeseeable changes, contractual issues, cargo readiness and/or according Terminal's convenience.,,

Be guided that Port Authority will provide the exchange of ship's fenders along shed 101 at the begining of the second half of August (agw-wp). Such procedure will take about 3 days and no vessels may be berthed.,,

"Weather Forecast: Tide Table: Aerial view of Grain Pier #101:",""


"Sao Francisco do Sul Weather Today (1-3 days) Sao Francisco do Sul Weather (4-7 days) 10 Day Sao Francisco do Sul Weather (7-10 days) Date Time Height",""


"Moderate rain (total 11mm), heaviest on Fri afternoon. Mostly dry. Warm (max 22°C on Mon afternoon, min 13°C Mostly dry. Warm (max 26°C on Wed afternoon, min 15°C 26-Jul 05:51 0.5",""


"Warm (max 25°C on Wed afternoon, min 14°C on Fri on Sat night). Wind will be generally light. on Tue morning). Wind will be generally light. 10:08 1.2",""


"night). Wind will be generally light. 14:00 0.8",""


"°C Wednesday Thursday Friday Saturday Sunday Monday 16:23 Tuesday 0.9 Wednesday Thursday 26 27 28 29 30 31 01 02 03 19:43 0.6",""


"°F AM PM Night AM PM Night AM PM Night AM PM Night AM PM Night AM PM Night AM PM Night AM PM Night AM PM Night 23:49 1.2",""


"27-Jul 06:09 0.3",""


"1,1 11:00 1.3",""


"14:58 0.9 Wind 0 15 5 0 0 10 5 10 5 10 0 16:47 0.9",""


km/h,,

"rain rain light light light light mod. clear light light light some some 20:28 0.5 clear clear clear clear clear clear clear rain clear clear clear clear shwrs rain clear shwrs rain rain rain rain rain rain clouds clouds 28-Jul 00:47 1.3",""


"06:45 0.2",""


"11:56 1.4",""


"See all 16:53 1.0 maps",""


"21:09 0.4",""


"a 29-Jul 01:32 1.4",""


"Rain 07:19 0.0 0.1 0.3 0.1 0.1 2.2 1.2 5 1.8 0.7 0.1 mm 12:58 1.5",""


"High 25 25 25 17 17 16 17 17 15 14 14 14 17 18 16 20 22 18 20 22 19 25 26 20 25 26 18 21:58 0.3",""


"Low 19 24 17 16 16 15 16 15 14 14 14 13 14 17 15 16 19 15 15 20 18 19 22 18 20 20 17 30-Jul 02:15 1.5",""


"07:56 Feels -0.1 25 25 26 15 16 17 18 16 13 13 13 14 16 18 16 21 22 20 21 22 20 25 26 21 26 25 19 14:02 1.6",""


"Humid 66 46 60 89 86 88 100 99 89 83 81 84 91 73 99 93 67 100 100 65 83 68 48 84 65 49 98 22:47 0.4",""

