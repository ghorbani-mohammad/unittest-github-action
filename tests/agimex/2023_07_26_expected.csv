report_date,contributor_lineup_reference_id,commodity_name,commodity_specification_codes,volume,unit,origin,destination,expected_cargo_loading_date,cargo_loading_date,cargo_discharged_date,cargo_status,carrier_id,carrier_name,carrier_status,carrier_dead_weight,export_location,import_location,export_terminal,import_terminal,eta,etb,etd,ata,atb,atd,agent,charterer,shipper,operator,comments
None,None,SBOIL,None,26300.0,MT,None,None,None,2023-07-26T09:45:00+00:00,None,LOAD,None,TORM BIRGITTE,None,None,Sao Francisco Do Sul,None,BA,None,None,None,2023-07-28T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,SBS,None,67200.0,MT,None,None,None,2023-07-28T00:00:00+00:00,None,LOAD,None,BULK GREECE,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-07-30T09:00:00+00:00,None,None,None,Alphamar,C&D Xiamen,None,Zport,None
None,None,CORN,None,67600.0,MT,None,None,None,2023-07-30T09:00:00+00:00,None,LOAD,None,BBG LIUZHOU,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-01T09:00:00+00:00,None,None,None,Cargonave,Cofco,None,Zport,None
None,None,SBS,None,66000.0,MT,None,None,None,2023-08-01T09:00:00+00:00,None,LOAD,None,PANAGIA FORCE,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-03T09:00:00+00:00,None,None,None,Cargonave,C.A.O.,None,Zport,None
None,None,SBS,None,66380.0,MT,None,None,None,2023-08-03T09:00:00+00:00,None,LOAD,None,SOYA TIANJIN,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-05T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,SBS,None,65000.0,MT,None,None,None,2023-08-05T09:00:00+00:00,None,LOAD,None,KIM OLDENDORFF,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-07T09:00:00+00:00,None,None,None,Cargonave,Cofco,None,Litoral,None
None,None,SBS,None,55000.0,MT,None,None,None,2023-08-07T09:00:00+00:00,None,LOAD,None,CALYPSO V,None,None,Sao Francisco Do Sul,None,TG,None,None,None,2023-08-09T09:00:00+00:00,None,None,None,Alphamar,BTG Pactual,None,Soin,None
None,None,SBS,None,68250.0,MT,None,None,None,2023-08-09T09:00:00+00:00,None,LOAD,None,ANGLO BARINTHUS,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-11T09:00:00+00:00,None,None,None,Cargonave,Olam,None,Zport,None
None,None,CORN,None,23000.0,MT,None,None,None,2023-08-11T09:00:00+00:00,None,LOAD,None,GREEN SAPHIRE,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-12T09:00:00+00:00,None,None,None,Alphamar,Enerfo,None,Zport,None
None,None,SBS,None,66000.0,MT,None,None,None,2023-08-12T09:00:00+00:00,None,LOAD,None,KAPTA DIMITRIS A,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-14T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,CORN,None,65000.0,MT,None,None,None,2023-08-14T09:00:00+00:00,None,LOAD,None,GENEVA STAR,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-16T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,CORN,None,67600.0,MT,None,None,None,2023-08-19T09:00:00+00:00,None,LOAD,None,BBG BRIGHT,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-21T09:00:00+00:00,None,None,None,Cargonave,Cofco,None,Zport,None
None,None,CORN,None,62000.0,MT,None,None,None,2023-08-21T09:00:00+00:00,None,LOAD,None,BLUE AKIHABARA,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-23T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,CORN,None,66000.0,MT,None,None,None,2023-08-23T09:00:00+00:00,None,LOAD,None,KEY JOURNEY,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-25T09:00:00+00:00,None,None,None,Cargonave,Cofco,None,Zport,None
None,None,SBS,None,66000.0,MT,None,None,None,2023-08-25T09:00:00+00:00,None,LOAD,None,WISDOM DIVA,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-27T09:00:00+00:00,None,None,None,Alphamar,C.A.O.,None,Zport,None
None,None,CORN,None,65000.0,MT,None,None,None,2023-08-27T09:00:00+00:00,None,LOAD,None,BRAVEHEART I,None,None,Sao Francisco Do Sul,None,BA+TG,None,None,None,2023-08-29T09:00:00+00:00,None,None,None,Litoral,Bunge,None,Litoral,None
None,None,SBS,None,66000.0,MT,None,None,None,2023-08-29T09:00:00+00:00,None,LOAD,None,SAKIZAYA BRAVE,None,None,Sao Francisco Do Sul,None,MG,None,None,None,2023-08-31T09:00:00+00:00,None,None,None,Cargonave,Cofco,None,Zport,None
